documentation/decent_user_interface.md

```diff
@@ -1721,7 +1726,7 @@ A few **dui add** commands just offer convenience shorthands to other commands,
 
 >Return the list of all canvas IDs that form the scale compound. The command also adds (if applicable) the following named tags to the canvas, and the same keys in the widgets page namespace array: 
 
-> >&lt;main_tag&gt;: the slider circle;
+> >&lt;main_tag&gt;-crc: the slider circle;

```

Comment:
?

Reply:
That is the actual tag of the circle, "<main_tag>-crc". I reviewed the code and the documentation didn't match the actual tag.